pkgs/tools/misc/phpDocumentor/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, makeWrapper, lib, php }:
+let
+  pname = "phpDocumentor";
+  version = "3.3.1";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/phpDocumentor/phpDocumentor/releases/download/v${version}/phpDocumentor.phar";
+    sha256 = "SpPSeP1FgfF3YJAxNNhfzePUDZP3OcjGSPPtAsnD57s=";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
The `preInstall`  and `postInstall` hooks are missing.

See: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/php-packages/grumphp/default.nix

Reply:
Fixed.